core/metacling/src/TCling.cxx

```diff
@@ -1283,6 +1297,10 @@ TCling::TCling(const char *name, const char *title, const char* const argv[])
                                          &(interpArgs[0]),
                                          llvmResourceDir);
 
+   fPedanticInterp = new cling::Interpreter(interpArgs.size(),
```

Comment:
Can we not create a second interpreter, but tweak the single interpreter that `TCling` is using? You'll have to get the sequencing right (the tweaking needs to be known at construction) - but that's still a lot better than bootstrapping two whole "cling/clang/llvm everythings"